controllers/C_Document.class.php

```diff
@@ -125,6 +126,8 @@ function upload_action_process() {
                     if ($_FILES['file']['size'][$key] == 0) {
                         $error .= "The system does not permit uploading files of with size 0.\n";
                     }
+                }elseif($GLOBALS['secure_upload'] && !isWhiteFile($_FILES['file']['tmp_name'][$key], $white_list)){
+                       $error = "The system does not permit uploading files with MIME content type - " . mime_content_type($_FILES['file']['tmp_name'][$key]) . ".\n";
```

Comment:
translate:
``````
$error = xl("The system does not permit uploading files with MIME content type") . " - " . mime_content_type($_FILES['file']['tmp_name'][$key]) . ".\n";